App/Controllers/Tag.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace App\Controllers;
+
+use App\Models\CategoryModel;
+use App\Models\PostModel;
+use App\Models\TagModel;
+use Core\Controller;
+use Core\Container;
+
+class Tag extends Controller
+{
+
+    protected $siteConfig;
+
+    public function __construct(Container $container)
+    {
+        $this->loadModules[] = 'SiteConfig';
+        parent::__construct($container);
+    }
+
+    /**
+     * @param int $tagId
+     * @param string $page
+     * @throws \ErrorException
+     * @throws \ReflectionException
+     * @throws \Twig_Error_Loader
+     * @throws \Twig_Error_Runtime
+     * @throws \Twig_Error_Syntax
+     */
+    public function posts(int $tagId, string $page = "page-1")
+    {
+        $postModel = new PostModel($this->container);
+        $tagModel = new TagModel($this->container);
```

Comment:
inject in your controller

Reply:
Resolved in branch Security_and_bug_fixes